proto/zitadel/user/v2/user_service.proto

```diff
@@ -2094,6 +2094,7 @@ message RetrieveIdentityProviderIntentResponse{
       example: "\"163840776835432345\"";
     }
   ];
+  AddHumanUserRequest add_human_user = 4;
```

Comment:
I don't get where the field is filled in the code

Reply:
https://github.com/zitadel/zitadel/pull/9420/files#diff-57ff532251c4a59a2bffb8938f4c0f3dde8111a0ac0c251ff05b60f204b2a037R197
The idea is that we move the mapping to the backend, so that we can cover the full functionality of the old actions.